asserts/header_checks.go

```diff
@@ -116,13 +116,17 @@ func checkIntWithDefault(headers map[string]interface{}, name string, defl int)
 	if !ok {
 		return -1, fmt.Errorf("%q header is not an integer: %v", name, value)
```

Comment:
this message needs also to be changed to use what

Reply:
Fixed!